.github/workflows/ci.yaml

```diff
@@ -249,12 +249,12 @@ jobs:
           echo PATH=$PATH >> $GITHUB_ENV
       - name: Install requirements
         run: |
-          python -m pip install --upgrade pip wheel
-          pip install -e ".[export]" pytest mlflow pycocotools
+          pip install uv
```

Comment:
@lakshanthad I think we have a github setup to install and set up `uv` package, instead of installing it through pip?
https://github.com/ultralytics/ultralytics/blob/e0f8eda366c7cf08a1c311a17eaaaf87914f8012/.github/workflows/ci.yaml#L55
for consistency perhaps we should the same here.

Reply:
Good catch! @Laughing-q. I have changed according to that now and let's see how the CI goes!